package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bs-react-testing-library",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Revert this as it is done when publishing to npm.

Reply:
Done.